utilities/user_utilities.py

```diff
@@ -494,9 +515,15 @@ def put_user_event(eventId):
             # Generation of URL via platformEventId
             url = current_app.config['EVENT_BUILDING_BLOCK_URL'] + '/' + event.get('platformEventId')
             # Getting rid of platformEventId from PUT request
+            platform_event_id = event["platformEventId"]
             if "platformEventId" in event:
                 del event["platformEventId"]
 
+            # get previous groupid from events building block
+            result = requests.get(url, headers={"ROKWIRE-API-KEY": Config.ROKWIRE_API_KEY})
```

Comment:
Maybe change the header to Bearer Token, as the group/event can be private. Hence The rokwire-api-key GET request would return 401.

Reply:
```suggestion
            result = requests.get(url, headers=headers)
```

Reply:
Also, to add more context, when such errors occur, the group change does not happen in the code starting at line 546 [below](https://github.com/rokwire/events-manager/pull/951/files#diff-189f38ad67f73611244c379b9c0d697430c65c8245a5841d08ada77624bc08eeR546), but no feedback is getting displayed for the user.